mix.lock

```diff
@@ -1,13 +1,18 @@
+  "db_connection": {:hex, :db_connection, "1.1.2", "2865c2a4bae0714e2213a0ce60a1b12d76a6efba0c51fbda59c9ab8d1accc7a8", [:mix], [{:connection, "~> 1.0.2", [hex: :connection, optional: false]}, {:poolboy, "~> 1.5", [hex: :poolboy, optional: true]}, {:sbroker, "~> 1.0", [hex: :sbroker, optional: true]}]},
+  "decimal": {:hex, :decimal, "1.4.0", "fac965ce71a46aab53d3a6ce45662806bdd708a4a95a65cde8a12eb0124a1333", [:mix], []},
+  "earmark": {:hex, :earmark, "1.2.3", "206eb2e2ac1a794aa5256f3982de7a76bf4579ff91cb28d0e17ea2c9491e46a4", [:mix], []},
+  "ex_const": {:hex, :ex_const, "0.1.0", "7b04d0b65f00cfb8afc0b778bd58dec4804612a54f57122524ee097e9f2c5443", [:mix], []},
+  "ex_doc": {:hex, :ex_doc, "0.17.1", "39f777415e769992e6732d9589dc5846ea587f01412241f4a774664c746affbb", [:mix], [{:earmark, "~> 1.1", [hex: :earmark, optional: false]}]},
+  "ex_spirit": {:hex, :ex_spirit, "0.3.3", "89bd061c68a63b8a4321d768d87e918a7ba60f5a044e5961b14544d1f062901d", [:mix], []},
+  "exconstructor": {:hex, :exconstructor, "1.1.0", "272623a7b203cb2901c20cbb92c5c3ab103cc0087ff7c881979e046043346752", [:mix], []},
+  "html_entities": {:hex, :html_entities, "0.3.0", "2f278ffc69c3f0cbd5996628fef37cf922fa715f3e04b9f38924c8adced3f25a", [:mix], []},
+  "inch_ex": {:hex, :inch_ex, "0.5.5", "b63f57e281467bd3456461525fdbc9e158c8edbe603da6e3e4671befde796a3d", [:mix], [{:poison, "~> 1.5 or ~> 2.0 or ~> 3.0", [hex: :poison, optional: false]}]},
+  "makedown": {:hex, :makedown, "0.2.1", "3242ab50efe664eb3181daa79659736c271e4ef348a367ca420103b8c366a8ed", [:mix], [{:earmark, "~> 1.1", [hex: :earmark, optional: false]}, {:html_entities, "~> 0.3.0", [hex: :html_entities, optional: false]}, {:makeup, "~> 0.2.0", [hex: :makeup, optional: false]}]},
+  "makeup": {:hex, :makeup, "0.2.0", "558327c02cd4bc75e54d6aa2e31752c0a906eb4b40b392237921348c4ce455a9", [:mix], [{:ex_const, "~> 0.1.0", [hex: :ex_const, optional: false]}, {:ex_spirit, "~> 0.3.0", [hex: :ex_spirit, optional: false]}, {:exconstructor, "~> 1.1.0", [hex: :exconstructor, optional: false]}, {:html_entities, "~> 0.3.0", [hex: :html_entities, optional: false]}]},
+  "makeup_elixir": {:hex, :makeup_elixir, "0.2.0", "2a16854b9ecc44386241b03e2be50b83d0cdeac38c1cbeed1b16812238443f3a", [:mix], [{:makeup, "0.2.0", [hex: :makeup, optional: false]}]},
   "mariaex": {:git, "https://github.com/xerions/mariaex.git", "59d7ae0ab21212bf3434b573c07e4fd2385ec0c8", []},
-  "poison": {:hex, :poison, "3.1.0", "d9eb636610e096f86f25d9a46f35a9facac35609a7591b3be3326e99a0484665", [:mix], [], "hexpm"},
-  "poolboy": {:hex, :poolboy, "1.5.1", "6b46163901cfd0a1b43d692657ed9d7e599853b3b21b95ae5ae0a777cf9b6ca8", [], [], "hexpm"},
+  "poison": {:hex, :poison, "3.1.0", "d9eb636610e096f86f25d9a46f35a9facac35609a7591b3be3326e99a0484665", [:mix], []},
+  "poolboy": {:hex, :poolboy, "1.5.1", "6b46163901cfd0a1b43d692657ed9d7e599853b3b21b95ae5ae0a777cf9b6ca8", [:rebar], []},
```

Comment:
Please update your Hex version so you don't revert changes in the mix.lock.

If you refactored makeup to not use dependencies such as ex_const and exconstructor it would improve your project and lower the amount of debt introduced through dependencies when your project is used.

Reply:
> If you refactored makeup to not use dependencies such as ex_const and exconstructor it would improve your project and lower the amount of debt introduced through dependencies when your project is used.

Already completed by @josevalim at:  https://github.com/tmbb/makeup/pull/4  ^.^